src/pages/en/guides/images.md

```diff
@@ -135,35 +126,39 @@ Image files in your project's source directory can be imported in frontmatter an
 
 #### Remote Images
 
-Remote images require a full URL as the image `src`. Also, you must either provide `width` and `height`, or one of the dimensions plus the required `aspectRatio` to the `<Image />` component.
+To use a remote image, pass a full URL to the `<Image />`'s src attribute. `<Image />` won't infer dimensions and format from this remote file. You must provide the `format` to render the image with, and you must either provide `width` and `height` or one of the two dimensions plus an `aspectRatio`.
```

Comment:
```suggestion
To use a remote image, pass a full URL to the `<Image />`'s src attribute. `<Image />` won't infer dimensions and format from this remote file. You must provide the `format` to render the image with, and you must either provide both `width` and `height` or one of the two dimensions plus an `aspectRatio`. The `alt` attribute is also required.
```
Maybe the "both" is helpful for a longer sentence explanation? (I don't think we have to bullet point the two options, but we could. An explicit "both" could be a good compromise.)

Also, it feels like a good place to remind re: `alt` while we're explicitly mentioning the attributes that are required.

Reply:
If we mention `alt` here, we'd need to reference it in all of the other sections (local/remote for Image/Picture), otherwise it will seem like alt is only required here. I'd rather state it in the intro section if we want a reminder

Reply:
I decided to take your approach because the other sections call out required attributes!